app/components/topic-card/component.js

```diff
@@ -14,8 +15,8 @@ export default Ember.Component.extend({
   isUpvoter: Ember.computed('session.currentUser', 'topic.upvoters.[]', function isUpvoter() {
     return this.get('topic.upvoters').contains(this.get('session.currentUser'));
   }),
-  isVolunteer: Ember.computed('session.currentUser', 'topic.volunteers.[]', function isVolunteer() {
-    return this.get('topic.volunteers').contains(this.get('session.currentUser'));
+  talkByCurrentUser: Ember.computed('session.currentUser', 'topic.talkBy', function() {
+    return this.get('session.currentUser.username') === this.get('topic.talkBy.username');
```

Comment:
I think I'm going to leave this as is, ,with the minor adjustments you mentioned. I'll put an issue in to investigate the `Ember.computed.equal` thing.